chromium_src/components/remote_cocoa/app_shim/native_widget_ns_window_bridge.mm

```diff
@@ -3,8 +3,33 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "base/apple/scoped_objc_class_swizzler.h"
+#include "skia/ext/skia_utils_mac.h"
+
 #include "src/components/remote_cocoa/app_shim/native_widget_ns_window_bridge.mm"
 
+namespace {
+NSColor* __strong g_original_title_color = nil;
+NSColor* __strong g_brave_title_color = nil;
+}  // namespace
+
+@interface BrowserWindowFrame : NativeWidgetMacNSWindowTitledFrame
+@end
+
+@interface FakeBrowserWindowFrame : NSObject
+@end
+
+@implementation FakeBrowserWindowFrame
```

Comment:
Don't swizzle, remove `FakeBrowserWindowFrame` :)

Override via Forward Declaration of `[NSThemeFrame _currentTitleColor]` and Category on `BrowserWindowFrame` instead.

```ObjC
@interface NSThemeFrame(BraveTheme)
- (NSColor*)_currentTitleColor;
@end

@interface BrowserWindowFrame : NativeWidgetMacNSWindowTitledFrame
@end

@implementation BrowserWindowFrame(BraveTheme)
- (NSColor*)_currentTitleColor {
  return g_brave_title_color;
}

- (NSColor*)originalTitleColor {
  return [super _currentTitleColor];
}
@end
```